### PR TITLE
tests: replace "test-snapd-base-bare" with real "bare" base snap

### PR DESCRIPTION
**Pull request by mvo5**
So far we did test that snapd work with an empty base using the
test-snapd-base-bare snap. However we have a "real" bare base
snap. This PR switch the tests to use the real "bare" base snap
so that we get continuous testing for it.

### CODE REVIEW

File: tests/lib/snaps/test-snapd-requires-base-bare/meta/snap.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 name: test-snapd-requires-base-bare
-base: test-snapd-base-bare
+base: bare
```
Comment (zyga) [inline tests/lib/snaps/test-snapd-requires-base-bare/meta/snap.yaml]: Please include the URL of the bare base source code here.

File: tests/lib/snaps/test-snapd-busybox-static/snapcraft.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: 1.0
 summary: Statically linked busybox for test
 description: |
   Statically linked busybox for test useful for e.g. bare base testing
-base: test-snapd-base-bare
+base: bare
```
Comment (zyga) [inline tests/lib/snaps/test-snapd-busybox-static/snapcraft.yaml]: Please include the URL of the bare base source code here.

**Comment by mvo5** [comment]
Blocked until the store fixes the type of the "bare" base :/

**Comment by pedronis** [comment]
The bare snap type is fixed in the store now, this can be unblocked.

**Pull request merged**